NGLess/Main.hs

```diff
@@ -211,7 +211,9 @@ modeExec opts@DefaultMode{} = do
     (shouldOutput,odir) <- runNGLessIO "loading and running script" $ do
         updateNglEnvironment (\e -> e { ngleScriptText = ngltext })
         sc' <- runNGLess $ parsengless fname reqversion ngltext >>= maybe_add_print
-        updateNglEnvironment (\e -> e {ngleVersion = (nglVersion $ fromJust $ nglHeader sc')})
+        updateNglEnvironment (\e -> e {ngleVersion = case (nglHeader sc') of
```

Comment:
You can write this more simply

    fromMaybe (T.pack versionStr) (nglVersion <$> nglHeader sc')